src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -63,9 +51,7 @@ const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React
 
   if (postNotifItem?.rawJson?.comments?.length > 0) {
     firstComment = postNotifItem?.rawJson?.comments[0];
-    anonymousPostNotificationUserInfo = {
-      ...firstComment?.reaction?.data
-    };
+    anonymousPostNotificationUserInfo = {...firstComment?.reaction?.data};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The spread operator is now used directly when assigning `anonymousPostNotificationUserInfo`. This change should not affect the functionality, but it's worth noting.

```diff
- anonymousPostNotificationUserInfo = {
-  ...firstComment?.reaction?.data
- };
+ anonymousPostNotificationUserInfo = {...firstComment?.reaction?.data};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->